README.md

```diff
@@ -65,6 +65,11 @@ The database structure required for using the default Firestore
 implementation, is described in
 [Firestore docs](https://github.com/strvcom/android-research-chat-component/blob/master/firestore_structure.md).
 
+#### Extension methods
+There are two convenient extension methods for an Activity available to simplify work with opening a camera and gallery in case only the basic functionality is required:
+- `Activity.openCamera(imageUri: Uri)`
```

Comment:
It could be useful to add links pointing to the file that contains the methods. The user could just navigate there and see the implementation with the doc just on one click, e.g.:(https://github.com/strvcom/android-chat-component/blob/5d1303568169b8c5f9c1266c235e1d78286d9566/core/src/main/java/com/strv/chat/core/core/ui/extensions/activity.kt#L25)